src/index.ts

```diff
@@ -1,12 +1,79 @@
 import { Probot } from "probot";
+import fs from 'fs';
+const k8s = require('@kubernetes/client-node');
+
+const kc = new k8s.KubeConfig();
+kc.loadFromDefault();
+const k8sApi = kc.makeApiClient(k8s.CoreV1Api);
+const k8sContext = kc.getCurrentContext();
+const namespace = (() => {
+  if (k8sContext === 'inClusterContext') {
+    return fs.readFileSync('/var/run/secrets/kubernetes.io/serviceaccount/namespace', 'utf8');
+  } else {
+    return k8sContext.split('/')[0];
+  }
+})();
+console.log(`Operating in the namespace: ${namespace}`);
 
 export = (app: Probot) => {
+
+  // Respond to an issue opened
   app.on("issues.opened", async (context) => {
     const issueComment = context.issue({
       body: "Thanks for opening this issue!",
     });
     await context.octokit.issues.createComment(issueComment);
   });
+
+  // Respond to the GitHub app installation
+  app.on("installation.created", async (context) =>{
+
+    // Get the installation token  and expiry time from the installation
+    const appAuth: any  = await context.octokit.auth({ type: "installation" });
+    // console.log(appAuth);
+
+    const repos = context.payload.repositories;
+    // console.log(repos);
+    const orgName = repos[0].full_name.split("/")[0];
+    app.log.info(`Operating on the Github Org: ${orgName}`);
+
+    // Iterate over the list of repositories for .github repo
+    var repo_exist = false;
+    for (const repo of repos) {
+      // console.log(repo);
+      if (repo.name == ".github") {
+        repo_exist = true;
+        break;
+      }
+    };
+    console.log(repo_exist);
```

Comment:
You should be able to do something like:
```suggestion
    const repo_exist = Boolean(repos.find(r => r.name === ".github"))
```

Reply:
thanks for suggestion :+1: